reana-ui/src/pages/signin/Signin.js

```diff
@@ -59,9 +59,21 @@ export default function Signin() {
       <Segment>
         {config.cernSSO && (
           <>
-            <Button basic fluid size="large" onClick={handleClick}>
+            <Button basic fluid size="large" onClick={() => handleClick("cern_openid")}>
               Sign in with CERN Single Sign-On
             </Button>
+            {(config.localUsers || config.keycloakSSO) && (
+              <Divider section horizontal>
+                or
+              </Divider>
```

Comment:
I think it would look better not putting the divider between the CERN SSO and Keycloak SSO buttons, but only before the login form for local users. What do you think? 

This issue is very minor, as I do not expect both CERN SSO and Keycloak SSO to be enabled at the same time.

Reply:
How about just a divider without text? I think in our instance we'll allow both options.